src/graph_compute/update_function.h

```diff
@@ -2,8 +2,10 @@
 #define UPDATE_FUNCTION_H_
 
 #include <math.h>
-#include <stdint.h>
+//  #include <stdint.h>
```

Comment:
Please don't comment out code, including preprocessor directives. It should either be in or out.